sechub-shared-kernel/src/test/java/com/daimler/sechub/sharedkernel/configuration/SecHubConfigurationValidatorTest.java

```diff
@@ -100,8 +100,8 @@ public void empty_webconfig_is_rejected() throws Exception {
 
 		/* prepare */
 		SecHubWebScanConfiguration webscan = mock(SecHubWebScanConfiguration.class);
-		List<URI> list = new ArrayList<>();
-		when(webscan.getUris()).thenReturn(list);
+		URI uri = URI.create("");
```

Comment:
Again: I had never thought this would be possible - an empty string as an URI...
Good that you find this!